sale_commission/wizard/wizard_settle.py

```diff
@@ -114,6 +114,7 @@ def action_settle(self):
                                 )
                             )
                         settlement_ids.append(settlement.id)
+                    line.read(["amount"])  # force amount computation
```

Comment:
This seems weird and shouldn't be needed. What are the steps for needing this?

Reply:
You are right, but running tests was found error related amount computation that causing exception per validation in **_check_settle_integrity** method. I'll try to find out the reason in order to remove that line.

thanks

Reply:
removed and added amount field computation in tests